### DISCUSSION TITLE
Next.js warnings output to stderr

### DISCUSSION THREAD

**Discussion by pacocoursey** [vercel/next.js#41124] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Relevant packages:
      next: 12.3.0
      react: 17.0.1
      react-dom: 17.0.1

### How are you deploying your application? (if relevant)

GCP

### Describe the Bug

Next.js server warnings are being outputted to stderr, which makes many platforms (including Google Cloud and Datadog) treat them as severe errors, which litters our logs and makes it difficult to find real application-level logs. Could these go to stdout instead?

<img width="1174" alt="gcn8desz@2x" src="https://user-images.githubusercontent.com/34928425/193322485-f871f41a-0969-42d9-8a83-19ef4d56b3a0.png">

### Expected Behavior

Non-critical warnings to go to stdout.

### Link to reproduction

https://github.com/vercel/next.js/tree/canary/examples/with-docker

### To Reproduce

1. Build and start a Next.js app in GCP (https://github.com/vercel/next.js/tree/canary/examples/with-docker)
2. Notice that warning logs are going to stderr

**Comment by balazsorban44** [comment]
Hmm, we _are_ using `console.warn` for warnings:

https://github.com/vercel/next.js/blob/cfd58ae80e950aa3e4ac342bb09933d9cfbcc30c/packages/next/build/output/log.ts#L21-L23

Which apparently is just an alias for `console.error` in Node.js: https://nodejs.org/api/console.html#consolewarndata-args, hence it shows up as an error for you. :thinking:

**Reply by john-zaprite** [reply]
Can you confirm if this is true for the new 'ExperimentalWarning' flag for the Fetch API in Node v17.5+?

I'm getting a lot of Error logs for this in my Vercel console since a recent upgrade from v16 to v18.

Further reference:
https://github.com/nodejs/node/discussions/45820#discussioncomment-4373244
